<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JudgesDialog</class>
 <widget class="QDialog" name="JudgesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>193</width>
    <height>248</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="1">
      <widget class="QLineEdit" name="editJphone"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>裁判性别</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>裁判电话</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>裁判编号</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>裁判姓名</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>裁判年龄</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboJgender">
       <item>
        <property name="text">
         <string>男</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>女</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="editJname"/>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="editJudgeID"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="editJage"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="btnOK">
     <property name="text">
      <string>确  定</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/images/images/704.bmp</normaloff>:/images/images/704.bmp</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="btnClose">
     <property name="text">
      <string>取  消</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/images/images/706.bmp</normaloff>:/images/images/706.bmp</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnOK</sender>
   <signal>clicked()</signal>
   <receiver>JudgesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>JudgesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
